core/src/main/java/com/sequenceiq/cloudbreak/service/upgrade/sync/component/ImageReaderService.java

```diff
@@ -35,7 +35,7 @@
      * @param datalake true if the stack is a datalake
      * @return a set of ClouderaManagerProducts present on the image
      */
-    public Set<ClouderaManagerProduct> getParcels(Set<Image> statedImages, boolean datalake) {
+    Set<ClouderaManagerProduct> getParcels(Set<Image> statedImages, boolean datalake) {
```

Comment:
I think the public could stay here, no need to be so restrictive

Reply:
Actually this class is only used just by another class. What would be the benefit of making them public? Package private expresses that it is used only here, I think there is value in it.

Reply:
I don't think so, it can stay this way. I don't know how you check the code but usually I just navigate through idea and don't care if something is package private. It's just troublesome when refactoring and something is pkg private and you have to modify it because of moving it into another package.